code/clue.js

```diff
@@ -53,7 +178,46 @@ const pickKiller = () => {
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
```

Comment:
I know this looked like this when we got the code, but would be nice to have it on one line/choose a better place for the line break :-)